nomos-services/chain/chain-leader/src/relays.rs

```diff
@@ -0,0 +1,183 @@
+    >(
+        service_resources_handle: &OpaqueServiceResourcesHandle<S, RuntimeServiceId>,
+    ) -> Self
+    where
+        S: ServiceData,
+        <S as ServiceData>::Message: Send + Sync + 'static,
+        <S as ServiceData>::Settings: Send + Sync + 'static,
+        <S as ServiceData>::State: Send + Sync + 'static,
+        ClPool::Key: Send,
+        BlendService: nomos_blend_service::ServiceComponents,
+        BlendService::BroadcastSettings: Send + Sync,
+        <BlendService as ServiceData>::Message: Send + 'static,
+        SamplingNetworkAdapter:
+            nomos_da_sampling::network::NetworkAdapter<RuntimeServiceId> + Send + Sync,
+        SamplingStorage:
+            nomos_da_sampling::storage::DaStorageAdapter<RuntimeServiceId> + Send + Sync,
+        TimeBackend: TimeBackendTrait,
+        TimeBackend::Settings: Clone + Send + Sync + 'static,
+        RuntimeServiceId: Debug
+            + Sync
+            + Send
+            + Display
+            + 'static
+            + AsServiceId<BlendService>
+            + AsServiceId<
+                TxMempoolService<
+                    ClPoolAdapter,
+                    SamplingNetworkAdapter,
+                    SamplingStorage,
+                    ClPool,
+                    RuntimeServiceId,
+                >,
+            >
+            + AsServiceId<
+                DaSamplingService<
+                    SamplingBackend,
+                    SamplingNetworkAdapter,
+                    SamplingStorage,
+                    RuntimeServiceId,
+                >,
+            >
+            + AsServiceId<TimeService<TimeBackend, RuntimeServiceId>>
+            + AsServiceId<CryptarchiaService>,
+        CryptarchiaService: CryptarchiaServiceData<ClPool::Item>,
+    {
+        let blend_relay = service_resources_handle
+            .overwatch_handle
+            .relay::<BlendService>()
+            .await
+            .expect(
+                "Relay connection with nomos_blend_service::BlendService should
+        succeed",
+            );
+
+        let cl_mempool_relay = service_resources_handle
+            .overwatch_handle
+            .relay::<TxMempoolService<_, _, _, _, _>>()
+            .await
+            .expect("Relay connection with CL MemPoolService should succeed");
+
+        let sampling_relay = service_resources_handle
+            .overwatch_handle
+            .relay::<DaSamplingService<_, _, _, _>>()
+            .await
+            .expect("Relay connection with SamplingService should succeed");
+
+        let time_relay = service_resources_handle
+            .overwatch_handle
+            .relay::<TimeService<_, _>>()
+            .await
+            .expect("Relay connection with TimeService should succeed");
+
+        Self::new(blend_relay, cl_mempool_relay, sampling_relay, time_relay)
+    }
+
+    pub const fn blend_relay(&self) -> &BlendRelay<BlendService> {
+        &self.blend_relay
+    }
+
+    pub const fn cl_mempool_relay(
```

Comment:
not sure whether this is coming from the old thing, but I thought we got rid of `cl_` naming

Reply:
yeah, it can go away now, we only have a mempool